roles/fits/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-fits_version: "{{ '0.8.5' }}"
-fits_download_url: http://projects.iq.harvard.edu/files/fits/files
+fits_version: "{{ '1.6.0' }}"
```

Comment:
I think there was a reason we had pinned the version of FITS to 0.8.5. I don't know what it was, but if we still need FITS, we should ask about version constraints in the dev channel.